app/types/sqlalchemy.py

```diff
@@ -27,3 +30,21 @@ def process_result_value(self, value, dialect):
         if value is not None:
             value = value.replace(tzinfo=datetime.UTC)
         return value
+
+
+# Pre-configured field type for UUID primary key (see https://docs.sqlalchemy.org/en/20/orm/declarative_tables.html#mapping-whole-column-declarations-to-python-types)
+primary_key = Annotated[uuid.UUID, mapped_column(primary_key=True)]
```

Comment:
I think it may be a better practice to use CamelCase for this type.
See https://peps.python.org/pep-0008/#type-variable-names
```suggestion
PrimaryKey = Annotated[uuid.UUID, mapped_column(primary_key=True)]
```